providers/dns/exec/exec.go

```diff
@@ -117,10 +118,27 @@ func (d *DNSProvider) run(ctx context.Context, command, domain, token, keyAuth s
 
 	cmd := exec.CommandContext(ctx, d.config.Program, args...)
 
-	output, err := cmd.CombinedOutput()
-	if len(output) > 0 {
-		log.Println(string(output))
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		return fmt.Errorf("create pipe: %w", err)
+	}
+
+	cmd.Stderr = cmd.Stdout
```

Comment:
Oooh, that actually answers a question that I had in the back of my mind, namely how to "capture" both STDOUT and STDERR and "combine" them into a single stream. I'm glad I saw this.

Also, I didn't realize you were a maintainer. Thanks for merging this so quickly.